metricbeat/helper/module.go

```diff
@@ -227,6 +244,7 @@ func (m *Module) processEvents(events []common.MapStr, metricSet *MetricSet) ([]
 			eventFieldName: event,
 			"metricset":    metricSet.Name,
 			"module":       m.name,
+			"fields":       m.Config.Fields,
```

Comment:
It's also documented here in case you ever need to provide reference to others: https://godoc.org/github.com/elastic/beats/libbeat/publisher#Client